cmd/cmd_utils.go

```diff
@@ -704,12 +704,36 @@ func showUsageExample(cmd *cobra.Command, details string) {
 	u.PrintInvalidUsageErrorAndExit(errors.New(details), suggestion)
 }
 
+// filterStackNames filters out file paths and returns valid stack names
+func filterStackNames(stacksMap map[string]any, basePath string) []string {
+	stackNames := make([]string, 0)
+	for stackName := range stacksMap {
+		// Skip if it's a file path (starts with base path)
+		if basePath != "" && strings.HasPrefix(stackName, basePath+"/") {
```

Comment:
No, this is not right either. Stacks can exist anywhere, what we include and exclude is entirely config driven. We are not consulting that configuration, and deriving an implementation that doesn’t correspond to the documentation. We should have a centralized function for this and it relates to stacks not general utils, IMO.

Reply:
See https://atmos.tools/cli/configuration/stacks/